workflow-templates/release-please.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Release Please Workflow",
+  "description": "Manage package releases and versioning with automatic Changelogs and NPM publish.",
+  "iconName": "graasp-logo",
+  "categories": ["Automation", "Deployment", "Continuous integration"],
+  "filePatterns": ["package.json$"]
```

Comment:
Is the `$` sign on purpose? I thought it was a regex, but in which case it would mark the beginning of a line

Reply:
I think it comes from [this example](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow) and is there to ensure that the string ends with `package.json`.

Reply:
Ah yes my bad, i mixed up with the `^` operator 😅